upup/pkg/fi/ca.go

```diff
@@ -44,7 +44,7 @@ type KeystoreItem struct {
 type Keystore interface {
 	// FindKeypair finds a cert & private key, returning nil where either is not found
 	// (if the certificate is found but not keypair, that is not an error: only the cert will be returned)
-	FindKeypair(name string) (*pki.Certificate, *pki.PrivateKey, error)
+	FindKeypair(name string) (*pki.Certificate, *pki.PrivateKey, string, error)
```

Comment:
I'm a little worried that we might need a version for both cert & privatekey, but we'll see...

Reply:
Well your test is not working ... see your PR for comments.